integration-tests/api-gateway/e2e/e2e-happy-path.test.ts

```diff
@@ -128,17 +106,18 @@ describe("End to end happy path journey", () => {
     expect(tokenResponse.status).toEqual(200);
     const accessToken = token.access_token;
 
-    const credIssApiURL = `https://${publicApi}.execute-api.eu-west-2.amazonaws.com/${environment}/credential/issue`;
+    const credIssApiURL = `https://${publicApi}.execute-api.eu-west-2.amazonaws.com/${environment}/credential/issue-test`;
```

Comment:
Does this mean that the endpoint that's in prod will be untested until it's changed over? How come you've decided to do it like this rather than just change over the actual endpoint?